wsdlgo/encoder.go

```diff
@@ -895,6 +895,7 @@ func (ge *goEncoder) genValidator(w io.Writer, typeName string, r *wsdl.Restrict
 
 func (ge *goEncoder) genGoStruct(w io.Writer, d *wsdl.Definitions, ct *wsdl.ComplexType) error {
 	if ct.Abstract {
+		fmt.Fprintf(w, "type %s struct {}\n\n", name)
```

Comment:
I think we probably prefer to have an empty `interface{}` instead of `struct{}` here. It makes sense at least for requests, because it can be associated to an object that can be encoded by the xml Encoder. Don't know if it works for responses/decoder though.